### PR TITLE
gramps: update to 5.1.5

### PR DESCRIPTION
**Pull request by mustaqimM**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/gramps/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'gramps'
 pkgname=gramps
-version=5.1.4
-revision=2
+version=5.1.5
+revision=1
 build_style=python3-module
 hostmakedepends="pkg-config intltool python3"
```
Comment (paper42) [inline srcpkgs/gramps/template:6]:
```suggestion
hostmakedepends="pkg-config gettext python3"
```
and test if it generates the same files

File: srcpkgs/gramps/template [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 depends="gtk+3 python3-gobject python3-bsddb3"
```
Comment (paper42) [inline srcpkgs/gramps/template:7]:
```suggestion
depends="gtk+3 python3-gobject python3-bsddb3"
checkdepends="python3-pytest $depends python3-lxml python3-jsonschema"
```

File: srcpkgs/gramps/template [hunk 3]
```diff
@@ -11,7 +11,7 @@ license="GPL-2.0-or-later"
 homepage="https://gramps-project.org"
 changelog="https://gramps-project.org/blog/2020/08/gramps-5-1-3-released/"
```
Comment (paper42) [inline srcpkgs/gramps/template:12]:
```suggestion
changelog="https://github.com/gramps-project/gramps/releases"
```

File: srcpkgs/gramps/template (hunk 1 above)
Comment (mustaqimM) [reply srcpkgs/gramps/template:6]: Well it works, but at first glance for some reason it doesn't generate a desktop file and a few other files:

```zsh
/usr/share/applications/gramps.desktop
/usr/share/metainfo/gramps.appdata.xml
/usr/share/mime/packages/gramps.xml
/usr/share/mime-info/gramps.keys
```

Also I'm not able to test other languages in depth but I ran it like so:

```zsh
env LC_ALL=nl_BE.UTF-8 LANGUAGE="sv" gramps
```

And it seems to work fine.

Comment (paper42) [reply srcpkgs/gramps/template:6]: You are right, in that case we shouldn't switch to plain gettext for now.

File: srcpkgs/gramps/template (hunk 3 above)
Comment (paper42) [reply srcpkgs/gramps/template:12]: or you could use `https://gramps-project.org/blog/blog/`

**Comment by paper42** [comment]
please rebase this branch on top of master to make CI run

**Pull request closed without merging**